profile_library/shelly/Shelly EM/model.json

```diff
@@ -0,0 +1,28 @@
+{
+  "measure_description": "The measured device fw: 1.14.1-rc1, Eco Mode enabled, Wifi Client enabled, LED disabled, one measurement clamp connected. The result represents the average value of measurements collected from the measurement device API for the following scenarios: dumb device/baseline consumption (~4.9W), the the relay turned off, and the relay turned on.",
+  "measure_method": "manual",
+  "measure_device": "Shelly PM Mini gen3",
+  "measure_device_firmware": "20241011-114503/1.4.4-g6d2a586",
+  "measure_settings": {
+    "SAMPLE_COUNT": 200,
+    "SLEEP_TIME": 0.5
+  },
+  "name": "Shelly EM",
+  "aliases": [
+    "SHEM"
+  ],
+  "discovery_by": "entity",
+  "device_type": "smart_switch",
+  "only_self_usage": false,
```

Comment:
This needs to be `true`, right? as it's a smart switch with built-in energy meter.

Reply:
You wrote on discord:
> `only_self_usage` have to be false for smart_switch or smart_dimmer when you want the user to have the possibility to input fixed and linear configuration.

I was initially thinking, we wanted to give such an option. But now, I admit, it would be unnecessarily redundant to built-in energy meter. So, agree. it should be true.